torch/_torch_docs.py

```diff
@@ -5627,6 +5627,78 @@ def merge_dicts(*dicts):
             [[-0.8437,  0.1727, -0.1398]]])
 """.format(**common_args))
 
+add_docstr(torch.swapdims, r"""
+swapdims(input, dim1, dim2) -> Tensor
+
+Alias for :func:`torch.transpose`.
+
+Swaps the dimension of :attr:`input` at the position in :attr:`dim1`
+to the position in :attr:`dim2`.
+
+Args:
+    {input}
+    dim1 (int): Original position of the dim to move
+    dim2 (int): Destination position
+
+Examples::
+
+    >>> x = torch.tensor([[[0,1],[2,3]],[[4,5],[6,7]]])
+    >>> x
+    tensor([[[0, 1],
+            [2, 3]],
+
+            [[4, 5],
+            [6, 7]]])
+    >>> torch.swapdims(x, 0, 1)
+    tensor([[[0, 1],
+            [4, 5]],
+
+            [[2, 3],
+            [6, 7]]])
+    >>> torch.swapdims(x, 0, 2)
+    tensor([[[0, 4],
+            [2, 6]],
+
+            [[1, 5],
+            [3, 7]]])    
+""".format(**common_args))
+
+add_docstr(torch.swapaxes, r"""
+swapaxes(input, axis1, axis2) -> Tensor
+
+Alias for :func:`torch.transpose`.
+
+Swaps the dimension of :attr:`input` at the position in :attr:`axis1`
```

Comment:
This part of the documentation still needs to be deleted.

Reply:
> What would you think about aliasing them but adding a comment about how swapdims/swapaxes are analogous to NumPy's swapaxes function? Would that be the best of both worlds?

https://github.com/pytorch/pytorch/pull/46041#issuecomment-713005625

Isn't the plan to keep alias as well as some documentation explaining this behaviour?

Reply:
What about deleting the redundant parts and then, after the sentence explaining its an alias, adding a new sentence like:

"This function is equivalent to Numpy's swapaxes function." 

?

Reply:
Makes sense.